FastSimulation/Configuration/python/DataMixerPreMix_cff.py

```diff
@@ -8,7 +8,7 @@
 import FastSimulation.Tracking.recoTrackAccumulator_cfi
 mixData.tracker = FastSimulation.Tracking.recoTrackAccumulator_cfi.recoTrackAccumulator.clone()
 mixData.tracker.pileUpTracks = cms.InputTag("mix","generalTracks")
-mixData.tracker.pileUpMVAValues = cms.InputTag("mix","generalTracksMVAVals")
+mixData.tracker.pileUpMVAValues = cms.InputTag("mix","generalTracksMVAValues")
```

Comment:
@VinInn 

If you happen to update your branch before merging,
please remove this line.